.github/actions/add-azure-application-insights-release-annotation/Add-ReleaseAnnotation.ps1

```diff
@@ -17,7 +17,10 @@ $annotation = @{
 
 # Encoding parenthesis to prevent the request from failing.
 $ApplicationInsightsResourceId = $ApplicationInsightsResourceId.Replace('(', '%28').Replace(')', '%29')
-$body = (ConvertTo-Json $annotation -Compress) -replace '(\\+)"', '$1$1"' -replace "`"", "`"`""
+
+# Double escaping is not needed anymore, for more info see: https://github.com/Azure/azure-cli/issues/15529#issuecomment-1211884315
+$body = ConvertTo-Json $annotation -Compress
+
```

Comment:
OK, this is weird: Running the modified code on PS 7.4.1 (same as the runners) gives me the "Unsupported Media Type" error. If I revert the changes, then it works, so it's the opposite of the runners. Adding the content-type header (JSON) results in a bad request.

Reply:
I'm not sure what to do with that, tbh.

Reply:
Updated the API call (Invoke-AzRestMethod instead of Az CLI), because that works consistently with this code.

Reply:
Oohhh yeah, I've tested that also with the double escaped characters...